docs/blog/2021-12-12-log4j-zero-day-mitigation-guide.mdx

```diff
@@ -253,13 +262,12 @@ default and removes the message lookup feature.
 
 We recommend you upgrade, if possible.  For most people, this is the final and correct solution to the issue.
 
-:::caution Version 2.15.0 still may be vulnerable
-Log4j version `2.15.0` which was previously thought to be secure has been found to still have a [limited vulnerability](https://lists.apache.org/thread/83y7dx5xvn3h5290q1twn16tltolv88f), 
-that could result in a DOS (but not RCE), users must update to `>= 2.16.0`.
-:::
-
 :::caution Version 2.16.0 vulnerable to DOS
+~~Log4j version `2.15.0` which was previously thought to be secure has been found to still have a [limited vulnerability](https://lists.apache.org/thread/83y7dx5xvn3h5290q1twn16tltolv88f),
+that could result in a DOS (but not RCE), users must update to `>= 2.16.0`.~~
```

Comment:
Lets just say update to 2.17.0?  It's confusing to say "update to 16....okay now update to 17"

I think having a cohesive and clear message is way more valuable than having a changelog of whats happened to our blog post.